operator/v1/types_network.go

```diff
@@ -256,6 +256,40 @@ type KuryrConfig struct {
 	// size by 1.
 	// +optional
 	OpenStackServiceNetwork string `json:"openStackServiceNetwork,omitempty"`
+
+	// enableDebug set to true will enable debug logs in kuryr-kuberntes.
+	// +optional
+	EnableDebug *bool `json:"enableDebug,omitempty"`
+
+	// enablePortPrepopulation when true will make Kuryr prepopulate each newly created port
+	// pool with a minimum number of ports. Kuryr uses Neutron port pooling to fight the fact
+	// that it takes a significant amount of time to create one. Instead of creating it when
+	// pod is being deployed, Kuryr keeps a number of ports ready to be attached to pods.
+	// +optional
+	EnablePortPrepopulation *bool `json:"enablePortPrepopulation,omitempty"`
+
+	// poolMaxPorts sets a maximum number of free ports that are being kept in a port pool.
+	// If the number of ports exceeds this setting, free ports will get deleted. Setting 0
+	// will disable this. For more information about port pools see enablePortPrepopulation
+	// setting.
+	// +kubebuilder:validation:Minimum=0
+	// +optional
+	PoolMaxPorts *int `json:"poolMaxPorts,omitempty"`
+
+	// poolMinPorts sets a minimum number of free ports that should be kept in a port pool.
+	// If the number of ports is lower than this setting, new ports will get created and
+	// added to pool. For more information about port pools see enablePortPrepopulation
+	// setting.
+	// +kubebuilder:validation:Minimum=0
+	// +optional
+	PoolMinPorts *int `json:"poolMinPorts,omitempty"`
```

Comment:
Similar questions about not set, why zero cannot be a good default, and interaction with enablePortPrepopulation.

Reply:
Yeah, 0 would not make any sense here. I'll add info about the default.